libcodechecker/libhandlers/parse.py

```diff
@@ -278,20 +320,30 @@ def main(args):
             files = [os.path.join(input_path, file_name) for file_name
                      in file_names]
 
+        file_change = False
         for file_path in files:
-            report_stats = parse(file_path,
-                                 context,
-                                 metadata_dict,
-                                 suppress_handler,
-                                 skip_handler,
-                                 'print_steps' in args)
+            report_stats, f_change = parse(file_path,
+                                           context,
+                                           metadata_dict,
+                                           suppress_handler,
+                                           skip_handler,
+                                           'print_steps' in args)
+            if not file_change:
+                file_change = f_change
 
             severity_stats.update(Counter(report_stats.get('severity',
                                           {})))
             file_stats.update(Counter(report_stats.get('files', {})))
             report_count.update(Counter(report_stats.get('reports', {})))
 
         print("\n----==== Summary ====----")
+        if file_change:
+            LOG.warning("The source file content changed since the last "
```

Comment:
One warning call with multiple lines?

Reply:
done

Reply:
* Maybe `Some source file contents are changed`.
* I think you shouldn't break the line.
* There is an extra white space before the world `skipped`.